src/v2/Components/Artwork/Details.tsx

```diff
@@ -230,14 +233,20 @@ export const Details: React.FC<DetailsProps> = ({
             )}
         </Flex>
       )}
+      {(!!rest.artwork.is_saved || isHovered) && (
```

Comment:
**question**: I think it is necessary to check `isTouch` and if it is `true`, then don't display the "save" button (this check is [performed](https://github.com/artsy/force/blob/main/src/v2/Components/Artwork/SaveButton/useSaveButton.tsx#L30) for `isSaveButtonVisible`). what do you think about it?

Reply:
Added that as well! 🤘